src/postamble.js

```diff
@@ -145,7 +145,6 @@ LDSO.init();
 #endif
 #endif
 
-/** @type {function(Array=)} */
```

Comment:
Maybe worth moving into the `#if`? But I'm not actually sure how these annotations help - does closure optimize better?

Reply:
The annotation is there so that the argument can be optional (the `=` after the argument type).

Now that we have the explicit default argument there is no need for that annotation at all since closure can see that it has a default, and calling with zero args is fine.